src/postgres/commands/deletePostgresFunction.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { window } from 'vscode';
+import { DialogResponses, IActionContext } from "vscode-azureextensionui";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../utils/localize";
+import { PostgresFunctionTreeItem } from "../tree/PostgresFunctionTreeItem";
+
+export async function deletePostgresFunction(context: IActionContext, treeItem?: PostgresFunctionTreeItem): Promise<void> {
+    if (!treeItem) {
+        treeItem = <PostgresFunctionTreeItem>await ext.tree.showTreeItemPicker(PostgresFunctionTreeItem.contextValue, context);
```

Comment:
Instead of `context`, do this:
```
{ ...context, suppressCreatePick: true }
```

To prevent someone from creating then instantly deleting